bayesian/api_v1.py

```diff
@@ -274,6 +274,17 @@ def get(ecosystem, package, version):
                                                                                      package=package, version=version)
             raise HTTPError(404, msg)
 
+    @staticmethod
+    def post(ecosystem, package, version):
+        decoded = decode_token()
+        if ecosystem == 'maven':
+            package = MavenCoordinates.normalize_str(package)
+        package = case_sensitivity_transform(ecosystem, package)
+
+        server_create_analysis(ecosystem, package, version,
+                               user_profile=decoded or {}, api_flow=True, force=True, force_graph_sync=False)
```

Comment:
@samuzzal-choudhury it looks like I'm getting error on local setup as there is no token, `decoded` is `None` and running the flow fails as there are not tracked user_profile information (thus `decoded or None` part). Is this intentional? Otherwise we would need to fix it on other places. WDYT?